packages/adapter-fauna/src/index.ts

```diff
@@ -73,11 +73,10 @@ const adapter = (
 				throw e;
 			}
 		}, setUser: async (userId, userData) => {
-			if (userId === null) throw new LuciaError("AUTO_USER_ID_GENERATION_NOT_SUPPORTED");
 			try {
 				const response: FaunaResponse = await faunaClient.query(q.Create(q.Collection("users"), {
 					data: {
-						id: userId,
+						id: userId || q.NewId(),
```

Comment:
Can you just change `||` to `??`. `userId` could be `0`

Reply:
The kysely, mongoose and primsa adapter all use `||`. From my understanding, their `undefined` request then generates the id in the DB. Shouldn't they be using `??` then as well?

Reply:
Thanks for pointing that out, yes it should be using `??`.